ui/src/services/swap.service.js

```diff
@@ -1,40 +1,35 @@
 import { E } from '@endo/captp';
-import { dappConfig } from '../utils/config.js';
 
 export const makeSwapOffer = async (
   walletP,
-  ammAPI,
   inputPurse,
   inputValue,
   outputPurse,
   outputValue,
 ) => {
-  console.log('PURSES:', inputPurse, outputPurse);
-  const id = `${Date.now()}`;
-  const { AMM_INSTALLATION_BOARD_ID, AMM_INSTANCE_BOARD_ID } = dappConfig;
-  const invitation = E(ammAPI).makeSwapInInvitation();
-
+  // TODO: Get this a better way?
+  const AMM_INSTANCE_BOARD_ID = 'board00917';
   const offerConfig = {
-    id,
-    invitation,
-    installationHandleBoardId: AMM_INSTALLATION_BOARD_ID,
+    invitationMaker: {
+      description: 'makeSwapInInvitation',
```

Comment:
this is the public facet `method` to call?

Reply:
Yes, I forgot to update it since we changed `description` to `method`. Done.